### ISSUE TITLE
More readable diagnostic output

### ISSUE THREAD

**Issue by dbarowy** [curtsinger-lab/riker#32]
Running `rkr why <file>` tells you, in plain English, why a file was regenerated during a build.  It can be combined with `--dry-run` to tell you what _would_ happen.

E.g.,

```
$ rkr why --dry-run <file>
```
produces output like
```
File 'file' was regenerated by 'command -abcd' because 'foo' changed.
```
where `foo` is the "root cause" of a rebuild (i.e., the file that changed).

Leaving off `<file>` produces a complete list of changes in English, ala `dodo check -vvv`, but in English.

**Comment by ccurtsinger** [comment]
You can view this information with `rkr --log rebuild`, although a more readable output may be useful in the future.
